client/state/data-layer/wpcom-http/pipeline/retry-on-failure/policies.js

```diff
@@ -1,8 +1,16 @@
 /** @format */
+
 export const exponentialBackoff = ( { delay = 1000, maxAttempts = 3 } = {} ) => ( {
 	name: 'EXPONENTIAL_BACKOFF',
 	delay: Math.max( 500, delay ),
 	maxAttempts: Math.min( 5, maxAttempts ),
+	allowedMethods: [ 'GET' ],
+} );
+
+export const exponentialBackoffExplicitPost = ( { delay, maxAttempts } = {} ) => ( {
+	...exponentialBackoff( delay, maxAttempts ),
+	allowedMethods: [ 'POST' ],
+	name: 'EXPONENTIAL_BACKOFF_EXPLICIT_POST',
```

Comment:
what if instead we used a wrapper to add `POST`? I feel that would be both more explicit and less redundant

```js
export const forPOST = policy => ( {
		...policy,
		allowedMethods: [ 'POST' ],
		name: `${ policy.name }-POST`
} );
```

```js
retryPolicy: forPOST( exponentialBackoff() )
```